go.mod

```diff
@@ -3,16 +3,30 @@ module github.com/openshift/cluster-api-provider-baremetal
 go 1.13
 
 require (
+	github.com/fatih/color v1.9.0 // indirect
+	github.com/gobuffalo/flect v0.2.1 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/json-iterator/go v1.1.10 // indirect
+	github.com/mattn/go-colorable v0.1.7 // indirect
 	github.com/metal3-io/baremetal-operator v0.0.0
 	github.com/onsi/gomega v1.9.0
 	github.com/openshift/machine-api-operator v0.2.1-0.20200721125631-d234cceb5de1
```

Comment:
This did not appear to update machine-api-operator depedency, actually.

Reply:
Excellent point :facepalm: